### PR TITLE
ScRNA tutorial updates 2022

### PR DESCRIPTION
**Pull request by nomadscientist**
Wherein I am updating all 3 (now 4) single cell tutorials with the latest tools
Also hoping to use this Github "File change' view to go through and do the same for the Spanish versions (although some text changes will be sketchy - would love help here!)

### CODE REVIEW

**Comment by nomadscientist** [comment]
- [ ] Alevin tutorial
- [ ] Combining datasets tutorial
- [ ] Filter, plot & explore tutorial
- [ ] Jupyter tutorial
- [ ] Spanish - Alevin
- [ ] Spanish - combining
- [ ] Spanish - filter
- [ ] Spanish - Jupyter

**Comment by nomadscientist** [comment]
Hi! I need these initial 3 tutorials sorted for testing this week in advance of training next week. Can I please get a review?

**Comment by hexylena** [comment]
Yep, I can review.

File: topics/transcriptomics/tutorials/scrna-case_alevin-combine-datasets/tutorial.md [hunk 1]
```diff
@@ -0,0 +1,321 @@
...
+>    - *"Expression matrix in sparse matrix format (.mtx)"*: `EBI SCXA Data Retrieval on E-MTAB-6945 matrix.mtx (Raw filtered counts)`
+>    - *"Gene table"*:  `EBI SCXA Data Retrieval on E-MTAB-6945 genes.tsv (Raw filtered counts)`
+>    - *"Barcode/cell table"*: `EBI SCXA Data Retrieval on E-MTAB-6945 barcodes.tsv (Raw filtered counts)`
+>    - *"Cell metadata table"*: `EBI SCXA Data Retrieval on E-MTAB-6945 exp_design.tsv`
+{: .hands_on}
+
+It's important to note that this matrix is processed somewhat through the SCXA pipeline, which is quite similar to this tutorial, and it contains any and all metadata provided by their pipeline as well as the authors (for instance, more cell or gene annotations).
+
+# Conclusion
+{:.no_toc}
```
Comment (hexylena) [inline topics/transcriptomics/tutorials/scrna-case_alevin-combine-datasets/tutorial.md:313]:
```suggestion
```

File: topics/transcriptomics/tutorials/scrna-case_alevin-combine-datasets/tutorial.md [hunk 2]
```diff
@@ -0,0 +1,321 @@
...
+
+translations:
+  - es
+
+gitter: Galaxy-Training-Network/galaxy-single-cell
+
+---
+
+# Introduction
+{:.no_toc}
```
Comment (hexylena) [inline topics/transcriptomics/tutorials/scrna-case_alevin-combine-datasets/tutorial.md:49]:
```suggestion
```

File: topics/transcriptomics/tutorials/scrna-case_alevin-combine-datasets/tutorial.md [hunk 3]
```diff
@@ -0,0 +1,321 @@
...
+> > 2. Under **Key-indexed observations annotation (obs)**. Different versions of the Manipulate tool will put the `batch` columns in different locations. The tool version in this course puts `batch` in the `8th` column. Batch refers to the order in which the matrices were added. The files are added from the bottom of the history upwards, so be careful how you set up your histories when running this (i.e. if your first dataset is N703 and the second is N701, the `batch` will call N703 `0` and N701 `1`!)
+> {: .solution}
+>
+{: .question}
+
+# Adding batch metadata
+
+I set up the example history with the earliest indices at the bottom.
+
+![Ordered history](../../images/scrna-casestudy/wab-history-files-ascending.png "Correct order of the history")
```
Comment (hexylena) [inline topics/transcriptomics/tutorials/scrna-case_alevin-combine-datasets/tutorial.md:154]: It looks like this alt-text doesn't describe the image very well for users who are relying on a screen reader to access the content. We have a short guide on [what's appropriate for alt-text vs captions in the GTN Contributing Guide](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/create-new-tutorial-content/tutorial.html#guidelines-on-alt-vs-figcaption-text) which has more details.

File: topics/transcriptomics/tutorials/scrna-case_alevin-combine-datasets/tutorial_ES.md [hunk 4]
```diff
@@ -213,9 +213,9 @@ Ahora tenemos:
 
 Ahora podemos ejecutar Alevin. Esta herramienta no aparecerá en las búsquedas dentro de algunas de las instancias públicas. Si esto sucede, haz click sobre la pestaña Single Cell en la parte izquierda de la pantalla y desplázate hacia abajo hasta encontrar la herramienta Alevin. Consejo: Si haces click sobre las herramientas (tools) dentro de la opción ‘tutorial’ de Galaxy siempre tendrás la versión correcta de la herramienta. En este caso: Galaxy Version 1.3.0+galaxy2 - debe ser la versión por defecto; si no lo es, haz click sobre ‘Versiones’ (‘Versions’) y escoge dicha versión.
 
-![Tutorial option in Galaxy](../../images/wab-tutorial-in-galaxy.png "La opción de tutorial está situada en la parte superior derecha de Galaxy")
+![Tutorial option in Galaxy](../../images/scrna-casestudy/wab-tutorial-in-galaxy.png "La opción de tutorial está situada en la parte superior derecha de Galaxy")
```
Comment (hexylena) [inline topics/transcriptomics/tutorials/scrna-case_alevin-combine-datasets/tutorial_ES.md:216]: this alt-text should probably be in spanish?

File: topics/transcriptomics/tutorials/scrna-case_alevin-combine-datasets/tutorial.md [hunk 5]
```diff
@@ -0,0 +1,321 @@
...
+
+tags:
+  - single-cell
+  - 10x
+  - paper-replication
+  - español
+
+contributors:
+  - nomadscientist
+  - pinin4fjords
```
Comment (hexylena) [inline topics/transcriptomics/tutorials/scrna-case_alevin-combine-datasets/tutorial.md:31]: If you like can use our fancy new contributions syntax to more accurately record contributions to a training material, declaring which people should be recorded for authorship or just editing.

```
contributions:
  authorship:
    - shiltemann
  editing:
    - bebatut
    - hexylena
  funding:
    - carpentries
  testing:
    - userX
  ux:
    - userY
  infrastructure:
    - userZ
```

see the [full gtn tutorial schema documentation](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/schemas/tutorial.html) for details on the individual roles.

Comment (hexylena) [reply topics/transcriptomics/tutorials/scrna-case_alevin-combine-datasets/tutorial.md:31]: And oops! I clearly need to add a "translator" role!

**Review by hexylena — COMMENTED** [review]
Overall this looks good!

the missing redirect is important, the rest, ok looks fine!

File: topics/transcriptomics/tutorials/scrna-case_JUPYTER-trajectories/tutorial.md [hunk 6]
```diff
@@ -1,10 +1,10 @@
 ---
 layout: tutorial_hands_on
```
[the remark this answers is not in the document]
Comment (nomadscientist) [reply topics/transcriptomics/tutorials/scrna-case_JUPYTER-trajectories/tutorial.md:3]: hang on this is a learning point for me - why does it need a redirect there?

[the remark this answers is not in the document]
Comment (nomadscientist) [reply topics/transcriptomics/tutorials/scrna-case_JUPYTER-trajectories/tutorial.md:3]: (I renamed a few things so will likely have to repeat this!)

File: topics/transcriptomics/tutorials/scrna-case_alevin-combine-datasets/tutorial.md [hunk 7]
```diff
@@ -0,0 +1,321 @@
...
+
+
+## Get Data
+The sample data is a subset of the reads in a mouse dataset of fetal growth restriction {% cite Bacon2018 %} (see the study in Single Cell Expression Atlas [here](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-6945/results/tsne) and the project submission [here](https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-6945/)). Each of the 7 samples (N701 --> N707) has been run through the workflow from the [Alevin tutorial](https://humancellatlas.usegalaxy.eu/training-material/topics/transcriptomics/tutorials/scrna-case_alevin/tutorial.html)
+
+You can access the data for this tutorial in multiple ways:
+
+ 1.  **Your own history** - If you're feeling confident that you successfully ran a workflow on all 7 samples from the previous tutorial, and that your resulting 7 AnnData objects look right (you can compare with the [answer key history](https://usegalaxy.eu/u/wendi.bacon.training/h/cs2combining-datasets-after-pre-processing---input-1)), then you can use those! To avoid a million-line history, I recommend dragging the resultant datasets into a fresh history
+ >
+ > {% snippet faqs/galaxy/histories_copy_dataset.md %}
```
[the remark this answers is not in the document]
Comment (nomadscientist) [reply topics/transcriptomics/tutorials/scrna-case_alevin-combine-datasets/tutorial.md:72]: It looks to be rendering ok so I think it's fine 
Screenshot 2022-09-08 at 12 35 13

File: topics/transcriptomics/tutorials/scrna-case_JUPYTER-trajectories/tutorial.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (nomadscientist) [reply topics/transcriptomics/tutorials/scrna-case_JUPYTER-trajectories/tutorial.md:3]: OHHHHHHh smart ok

File: topics/transcriptomics/tutorials/scrna-case_alevin-combine-datasets/tutorial.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (hexylena) [reply topics/transcriptomics/tutorials/scrna-case_alevin-combine-datasets/tutorial.md:72]: Yeah it's in an unstyled blockquote, which is invisible, and not really semantically useful to have in there. 

```suggestion

   {% snippet faqs/galaxy/histories_copy_dataset.md %}
```

Just removing that, and indenting it to match the first `*` of  `1. **Your own`

File: topics/transcriptomics/tutorials/scrna-case_JUPYTER-trajectories/tutorial.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (nomadscientist) [reply topics/transcriptomics/tutorials/scrna-case_JUPYTER-trajectories/tutorial.md:3]: Think I did it but also had to rebase on Git which is sketchy for me but let's say it's fine

File: topics/transcriptomics/tutorials/scrna-case_alevin-combine-datasets/tutorial.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (nomadscientist) [reply topics/transcriptomics/tutorials/scrna-case_alevin-combine-datasets/tutorial.md:72]: Well obviously it's an unsettled block quote, what kind of newb would put that in? What a chump

**Comment by nomadscientist** [comment]
OK all sorted I think? :)

**Review by hexylena — APPROVED** [review]

**Pull request merged**